.env.template

```diff
@@ -109,6 +109,36 @@ MODELS=`[
         "max_new_tokens": 2048,
         "stop": ["</s>"]
       },
+      "promptExamples": [
+        {
+          "title": "Write an email from bullet list",
+          "prompt": "As a restaurant owner, write a professional email to the supplier to get these products every week: \n\n- Wine (x10)\n- Eggs (x24)\n- Bread (x12)"
+        }, {
+          "title": "Code a snake game",
+          "prompt": "Code a basic snake game in python, give explanations for each step."
+        }, {
+          "title": "Assist in a task",
+          "prompt": "How do I make a delicious lemon cheesecake?"
+        }
+      ]
+    },
+        {
+      "name": "HuggingFaceM4/idefics-80b-instruct",
```

Comment:
We can remove this change to .env.template if we don't want IDEFICS in production for HuggingChat

Reply:
cc @julien-c to confirm this